cachito/web/content_manifest.py

```diff
@@ -21,17 +21,17 @@ def __init__(self, request):
         :param Request request: the request to generate a ContentManifest for
         """
         self.request = request
-        # dict to store go package level data; uses the package purl as key to identify a package
+        # dict to store go package level data; uses the package id as key to identify a package
```

Comment:
@chmeliik  Oh, looks like we will need to change integration tests that check content manifest. Because they keep old key data, am I right?

Reply:
Integration tests should be fine, the change only affects purls for local dependencies (we do not test those). Content manifests for everything else should stay exactly the same.

Also, I ran `tox -e integration` and it worked, so it should be fine :D

Reply:
Perfect, thank you!